kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/extensions/KroxyliciousExtension.java

```diff
@@ -42,7 +43,8 @@ public KroxyliciousExtension() {
 
     @Override
     public boolean supportsParameter(ParameterContext parameterContext, ExtensionContext extensionContext) throws ParameterResolutionException {
-        return !parameterContext.getParameter().getType().isAssignableFrom(TestInfo.class);
+        return !parameterContext.getParameter().getType().isAssignableFrom(TestInfo.class)
+                && !parameterContext.getParameter().getType().equals(TestKmsFacade.class);
```

Comment:
this looks weird.  flip it about so the KroxyliciousExtension knows what it wants rather than what it doesn't.   It shouldn't need to know about TestKmsFacade